### PR TITLE
LPD-34415 Fixing some Commerce order freemarker notification template terms that wasn't working

### PR DESCRIPTION
**Pull request by victorKammerer**
[LPD-34415](https://liferay.atlassian.net/browse/LPD-34415)

[LPD-34415]: https://liferay.atlassian.net/browse/LPD-34415?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by victorKammerer** [comment]
This commit will be added on @devpedrotavares  [PR (#3606)](https://github.com/liferay-objects/liferay-portal/pull/3606)

**Review by MarinhoFeliphe — APPROVED** [review]
LGTM!! :tada:

@victorKammerer good job, it is safe, I'm not sure if we have to send this to commerce team I believe we can just notify them.

File: modules/apps/commerce/commerce-order-content-web/src/main/java/com/liferay/commerce/order/content/web/internal/info/item/provider/CommerceOrderInfoItemFieldValuesProvider.java [hunk 1]
```diff
@@ -123,6 +123,12 @@ private List<InfoFieldValue<Object>> _getCommerceOrderInfoFieldValues(
 					_isOrderStatus(
 						commerceOrder,
 						CommerceOrderConstants.ORDER_STATUS_COMPLETED)));
+			commerceOrderInfoFieldValues.add(
+				new InfoFieldValue<>(
+					CommerceOrderInfoItemFields.completedOrderStatusInfoField,
+					_isOrderStatus(
+						commerceOrder,
+						CommerceOrderConstants.ORDER_STATUS_COMPLETED)));
```
Comment (MarinhoFeliphe) [inline modules/apps/commerce/commerce-order-content-web/src/main/java/com/liferay/commerce/order/content/web/internal/info/item/provider/CommerceOrderInfoItemFieldValuesProvider.java:131]: Hey @victorKammerer in the `InfoFieldValue` above we already have the map for `CommerceOrderConstants.ORDER_STATUS_COMPLETED`, I believe we can just replace `CommerceOrderInfoItemFields.companyIdInfoField` with `CommerceOrderInfoItemFields.completedOrderStatusInfoField`

File: modules/apps/commerce/commerce-order-content-web/src/main/java/com/liferay/commerce/order/content/web/internal/info/item/provider/CommerceOrderInfoItemFieldValuesProvider.java [hunk 2]
```diff
@@ -579,6 +585,10 @@ private List<InfoFieldValue<Object>> _getCommerceOrderInfoFieldValues(
 				new InfoFieldValue<>(
 					CommerceOrderInfoItemFields.totalDiscountAmountInfoField,
 					commerceOrder.getTotalDiscountAmount()));
+			commerceOrderInfoFieldValues.add(
+				new InfoFieldValue<>(
+					CommerceOrderInfoItemFields.totalWithTaxAmountInfoField,
+					commerceOrder.getTotalWithTaxAmount()));
```
Comment (MarinhoFeliphe) [inline modules/apps/commerce/commerce-order-content-web/src/main/java/com/liferay/commerce/order/content/web/internal/info/item/provider/CommerceOrderInfoItemFieldValuesProvider.java:591]: @victorKammerer this statement should came after the one related to `commerceOrder.getTotalDiscountWithTaxAmount()` to keep the asc sorting

**Review by MarinhoFeliphe — CHANGES_REQUESTED** [review]
Hey @victorKammerer please try to provide a test for this class covering only the fields you modified, see this example `JournalArticleInfoItemFieldValuesProviderTest`.

Thanks!

**Review by MarinhoFeliphe — APPROVED** [review]
LGTM!! :tada:

@victorKammerer please forward to commerce team and close this pull request.

Thanks!

**Comment by MarinhoFeliphe** [comment]
Resent here: https://github.com/liferay-commerce/liferay-portal/pull/5311

**Pull request closed without merging**